### DISCUSSION TITLE
Jólajól Biskupsreglunnar - 2022/11/6

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#24] [vefsida-ummæli]
# Jólajól Biskupsreglunnar - 2022/11/6

Biskupsreglan hélt upp á vor jól í skólanum síðast liðnu daga. Vér sungum, dösnsuðum, bökuðum, elduðum og margt fleira. Strákarnir okkar elska ekkert meir en vor jól.

https://www.biskupsreglan.com/frettir/jolajol

**Comment by TJ478** [comment]
Þetta voru stórksostlegustu jól lífs míns
